docs/sphinx/using/backends/dynamics.rst

```diff
@@ -33,8 +33,17 @@ Using CUDA-Q `operator`, the above time-dependent Hamiltonian can be set up as f
         :start-after: [Begin Transmon]
         :end-before: [End Transmon]
 
-In particular, `ScalarOperator` provides an easy way to model arbitrary time-dependent control signals.   
-Details about CUDA-Q `operator`, including builtin operators that it supports can be found :ref:`here <operators>`.
+  In particular, `ScalarOperator` provides an easy way to model arbitrary time-dependent control signals.
+  Details about CUDA-Q `operator`, including builtin operators that it supports can be found :ref:`here <operators>`.
+
+.. tab:: C++
```

Comment:
The Python tab codes include setting the target to dynamics - should we add the equivalent in the C++ tabs?